jvm/sargon-android/build.gradle.kts

```diff
@@ -46,7 +46,6 @@ android {
             java.srcDir("${buildDir}/generated/src/release/java")
         }
     }
-    packaging { resources.excludes.add("META-INF/*") }
```

Comment:
This fixed the issue of kotlin extensions not being visible.

Reply:
Great! @GhenadieVP @micbakos-rdx i think Michael said that Ghenadie added it. 

Any good reason for adding it Ghenadie?

Michael did your linking issues get fixed when you tried removing this?